code/game/gamemodes/endgame/supermatter_cascade/universe.dm

```diff
@@ -58,15 +58,11 @@ var/global/universe_has_ended = 0
 
 	spawn(rand(30,60) SECONDS)
 		var/txt = {"
-There's been a galaxy-wide electromagnetic pulse.  All of our systems are heavily damaged and many personnel are dead or dying. We are seeing increasing indications of the universe itself beginning to unravel.
+AUTOMATED ALERT: Attention [station_name()], this is a high alert broadcast to all ships from the central communication hub of the Hanza Trade Union, a catastrophe has happened on the ship [station_name()], information regarding the incident is classified.
 
-[station_name()], you are the only facility nearby a bluespace rift, which is near your research outpost. You are hereby directed to enter the rift using all means necessary, quite possibly as the last of your species alive.
-
-You have five minutes before the universe collapses. Good l\[\[###!!!-
+We highly suggest, that all corporate owned, and free ships within listening range depart into Bluespace. Until the incident ends, all pay-rolls will be frozen, and employees aboard currently employed HTU ships will have benefits cut.
```

Comment:
Eris is indeed one of the Hansa registered ships but it's still owned by captain and crew is paid by them and not by HTU, so it's probably best to remove the pay-roll part.

Reply:
I understand that it's probably an automated message sent to all ships but that just sounds weird

Reply:
I've added in an addition that independent ships are free from their payroll being cut, however I believe the players will have to have the brains to understand this, I do not them babied.